external/sql/storm-sql-core/src/test/org/apache/storm/sql/compiler/backends/trident/TestPlanCompiler.java

```diff
@@ -274,4 +274,9 @@ private void waitForCompletion(long timeout, Callable<Boolean> cond) throws Exce
       Thread.sleep(100);
     }
   }
+
+  private void assertArrayWithoutOrder(List<Values> expected, List<List<Object>> actual) {
+    Assert.assertTrue("Two arrays are not same!\n"+ "Expected: " + expected + "\n" + "Actual: " + actual,
```

Comment:
@satishd Thanks for the detailed review. Changed to `Two lists` :) also rename the method name to represent list.